java-checks/src/test/files/checks/AssertionsInTestsCheck/Junit4.java

```diff
@@ -51,7 +55,7 @@ public void junit_assert_true() {
 
   @Test
   public void junit_assert_that() {
-    org.junit.Assert.assertThat("aaa", org.junit.matchers.JUnitMatchers.containsString("a"));
+    assertThat("aaa", org.junit.matchers.JUnitMatchers.containsString("a"));
```

Comment:
Do not change that. Keep it explicit to show that it's a junit assert.

Reply:
I removed all changes on AssertionsInTestsCheck/Junit4.java